lib/data/hive/hive_collection_tom_database.dart

```diff
@@ -72,8 +73,11 @@ class HiveCollectionToMDatabase {
     try {
       await db.open();
     } catch (e, s) {
-      Logs().w('Unable to open Hive.', e, s);
+      Logs().w('Unable to open ToM Hive.', e, s);
+      const FlutterSecureStorage()
+          .delete(key: FlutterHiveCollectionsDatabase.cipherStorageKey);
       await db.clear().catchError((_) {});
+      await Hive.deleteFromDisk();
```

Comment:
deleteFromDisk() then is it retry open to continue ?

Reply:
back to old logic. we only handle this `clear()` func

Reply:
Any question @drminh2807